### DISCUSSION TITLE
LND Setup

### DISCUSSION THREAD

**Discussion by Scully78** [lightningnetwork/lnd#9186] [General]
I'm trying to set up LND but i keep getting a message when i try to load or create a wallet.
[lncli] rpc error: code = Unavailable desc = connection error: desc = "transport: Error while dialing: dial tcp 0.0.0.0 :10009: connect: connection refused".

I've tried altering the lnd.conf file with various suggestions I've found elsewhere but nothing seems to work.

**Comment by saubyk** [comment]
Hi, more details would be helpful here...
- What version of LND?
- What's your Bitcoin backend like and version?
- Are you setting up LND standalone or is it with LitD?
- What's your LND config like?
